ui/main/src/app/business/builtInTemplates/task/usercard/taskUserCardTemplate.ts

```diff
@@ -179,7 +179,7 @@ export class TaskUserCardTemplate extends BaseUserCardTemplate {
                         <td style="width: 90px">
                             <div class="opfab-input">
                                 <label> ${opfab.utils.getTranslation('builtInTemplate.taskUserCard.dayNumber')} </label>
-                                <input size="1" maxlength="2" type="number" id="nthDay" min="1" max="31" oninput="if (this.value > 31) this.value = '';">
+                                <input size="1" maxlength="2" type="number" id="nthDay" min="1" max="31" oninput="if ((this.value > 31) || (this.value < -31)) this.value = '';">
```

Comment:
I think we should set also min="-31" so it will be possible to set negative numbers using the arrows

Reply:
You're right.Done.